compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/SwingPanel.desktop.kt

```diff
@@ -194,67 +207,77 @@ private class FocusSwitcher<T : Component>(
     private val interopComponent: SwingInteropComponent<T>,
     private val focusManager: FocusManager,
 ) {
```

Comment:
I'd like to extract this class into separate file (with name something like `SwingInteropFocusSwitcher`) at some point. Not a blocker

Reply:
I wouldn't move it out of this file, until the file size is manageable (<1000 lines). Because this logic is strictly bound to `SwingPanel`.

A thing I would extract is `FocusTracker` - it might be useful for other platforms. But only when it is needed - there is a chance that other implementation should use the Android way.